src/test/java/uk/gov/hmcts/cmc/claimstore/services/content/FullAdmissionStaffEmailContentProviderTest.java

```diff
@@ -0,0 +1,94 @@
+package uk.gov.hmcts.cmc.claimstore.services.content;
+
+import org.junit.Before;
+import org.junit.Test;
+import uk.gov.hmcts.cmc.claimstore.config.PebbleConfiguration;
+import uk.gov.hmcts.cmc.claimstore.config.properties.emails.StaffEmailTemplates;
+import uk.gov.hmcts.cmc.claimstore.services.TemplateService;
+import uk.gov.hmcts.cmc.claimstore.services.staff.content.FullAdmissionStaffEmailContentProvider;
+import uk.gov.hmcts.cmc.claimstore.services.staff.models.EmailContent;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaim;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleResponse;
+
+import java.time.LocalDateTime;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.cmc.claimstore.services.staff.DefendantResponseStaffNotificationService.wrapInMap;
+
+public class FullAdmissionStaffEmailContentProviderTest {
+
+    private static final String DEFENDANT_EMAIL = "defendant@mail.com";
+
+    private TemplateService templateService = new TemplateService(
+        new PebbleConfiguration().pebbleEngine()
+    );
+
+    private StaffEmailTemplates templates = new StaffEmailTemplates();
+
+    private FullAdmissionStaffEmailContentProvider service;
+
+    @Before
+    public void beforeEachTest() {
+        service = new FullAdmissionStaffEmailContentProvider(
+            templateService,
+            templates
+        );
+    }
+
+    @Test
+    public void shouldUseRequiredFieldsInTheSubject() {
+        Claim claimWithFullAdmission = SampleClaim.builder()
+            .withResponse(SampleResponse.FullAdmission.builder().buildWithPaymentOptionImmediately())
+            .withRespondedAt(LocalDateTime.now())
+            .build();
+        EmailContent content = service.createContent(wrapInMap(claimWithFullAdmission, DEFENDANT_EMAIL));
+        assertThat(content.getSubject())
+            .contains("Pay immediately 000CM001:")
+            .contains("John Rambo v John Smith");
+    }
+
+    @Test
+    public void shouldUsePaymentOptionImmediatelyInTheBody() {
```

Comment:
This is a bit inconsistent, last two tests in the class are verifying both subject line and body while here we split it into two tests.